docs/documentation/upgrading/topics/changes/changes-26_2_0.adoc

```diff
@@ -48,6 +48,46 @@ If your deployment scripts add explicit NetworkPolicies for {project_name}, you
+
+* It is possible to have both the standard and the legacy token exchange features enabled, which can be useful if you need to cover standard use cases (internal-internal) together with the other token exchange use cases,
+which are implemented only by legacy token exchange. For instance, the link:{securing_apps_token_exchange_link}#_external-token-to-internal-token-exchange[external to internal token exchange] is implemented only by the 
+legacy token exchange. In this case {project_name} will serve standard internal-to-internal requests preferably by the standard token exchange when the other requests will be served by the legacy token exchange. It is detected based on the
+parameters if the particular request is standard or not. For instance requests containing non-standard parameters like `requested_issuer` or `requested_subject` are considered legacy.
+
+* Standard token exchange requires enabling a switch on the client as described in the link:{securing_apps_token_exchange_link}#_standard-token-exchange-enable[{securing_apps_token_exchange_name}].
+
+A few more clarifications regarding the notable changes in the behavior of the standard token exchange compared to the legacy token exchange:
+
+* Fine-grained admin permissions are no longer needed or supported for the standard token exchange.
+
+* The most notable change regarding the behavior of scopes and audiences is that the applied client scopes are based on the client triggering the token exchange request rather than the "target" client specified by the `audience` parameter.
+There is support for multiple values of the `audience` parameter as mentioned in the specification. The details are described in the link:{securing_apps_token_exchange_link}#_standard-token-exchange-scope[{securing_apps_token_exchange_name}].
+
+* Public clients are no longer allowed to send the token exchange requests. Legacy token exchange allowed public clients to exchange tokens with themselves to downscope the original token. This use case can be
+instead be covered by using the refresh token grant, in which the `scope` parameter can be used to downscope the refreshed access token, as mentioned in
+the https://datatracker.ietf.org/doc/html/rfc6749#section-6[OAuth2 specification].
+
+* Exchanging an access token for a SAML assertion is not supported for now. In other words, using `requested_token_type=urn:ietf:params:oauth:token-type:saml2` is not supported.
+
+* Exchanging an access token for a refresh token is allowed only if it is explicitly enabled on the client as mentioned in the link:{securing_apps_token_exchange_link}#_standard-token-exchange-details[{securing_apps_token_exchange_name}].
+Currently, it is not supported to request offline tokens or exchange a refresh token when the subject token was issued from an offline session. It is recommended to exchange for access tokens instead of
```

Comment:
```suggestion
Currently, it is not supported to request offline tokens or exchange a refresh token when the subject token was issued from an offline session. The recommended approach is to exchange for access tokens instead of
```